rules/opentitan_test.bzl

```diff
@@ -452,3 +454,138 @@ def opentitan_functest(
             # For more see https://bazel.build/reference/be/general#test_suite.tags
         ],
     )
+
+def _opentitan_sram_fpga_cw310_test(ctx):
+    test_script = '''#!/usr/bin/env bash
```

Comment:
What do you think about putting this in a separate file instead of inlining here?

Reply:
We could do that, WDYT about `rules/opentitan_gdb_test.bzl`?

Reply:
SGTM